src/components/dashboard/Claim.js

```diff
@@ -214,29 +189,81 @@ const Claim = props => {
         <Section.Stack style={styles.extraInfo}>
           <Image source={illustration} style={styles.illustration} resizeMode="contain" />
```

Comment:
illustration is bigger when there is only 1 button and almost no space between the illustration text and button

Reply:
> illustration is bigger when there is only 1 button and almost no space between the illustration text and button

@sirpy, yes. I focused on this to have the elements dispositions and logic.

Following the recomendation in https://help.zenhub.com/support/solutions/articles/43000010350-connecting-pull-requests-to-github-issues

> https://help.zenhub.com/support/solutions/articles/43000010350-connecting-pull-requests-to-github-issues

and was working on a separate PR for the #494 issue.

Reply:
@fernandomg please notice that this is not about smaller/larger screens.
Once you display only ONE BUTTON then the illustraion has more room and according to the design it should be larger doesn't matter if screen is smaller or larger

Reply:
oh ok i see.